unison-src/transcripts/names.md

```diff
@@ -1,20 +1,46 @@
- Example uses of the `names` command and output
+# `names` command
+
 ```ucm:hide
-.> alias.type ##Int .builtins.Int
+.> builtins.mergeio
```

Comment:
The reason for using `type.alias` despite looking ugly is that `builtins.mergeio` is (or at least was?) pretty slow, which adds up when you are running all the transcripts locally. At one point I/we made a pass to eliminate usages of `mergeio` where a small number of aliases for builtins would suffice.

Reply:
Hrmm I see.

My understanding would be that with the new version of the transcripts executable which avoids forking a new UCM for every transcript evaluation would be that we pay the `mergeio` cost once per call to `stack exec transcripts` rather than once per usage of `builtins.mergeio`, which means it's probably less costly than it used to be, but I haven't tested this or anything :shrug:

Reply:
Agreed 👍🏼 

Turns out that in this case I don't actually use any of the builtins anyways 😶

Reply:
Yeah. `##Nat` does show up in the output, so you could add an alias for that if you want, but it doesn't particularly bother me.